cli_tests/install_ketch.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+# exit when any command fails
+set -e
+# keep track of the last executed command
+trap 'last_command=$current_command; current_command=$BASH_COMMAND' DEBUG
+# echo an error message before exiting
+trap 'echo "\"${last_command}\" command filed with exit code $?."' EXIT
+
+# start cluster
+sudo minikube start --profile=minikube --vm-driver=none --kubernetes-version=v1.20.1
+sudo chown -R travis /home/travis/.minikube/
+
+# kubebuilder
+make install-kubebuilder KUBEBUILDER_INSTALL_DIR=/tmp/kubebuilder
+
+# kustomize
+make install-kustomize KUSTOMIZE_INSTALL_DIR=/tmp
+export PATH=$PATH:/tmp
+
+# ketch
+kubectl cluster-info
+make manifests install ketch manager
+export PATH=$PATH:$(pwd)/bin
+ketch -v
```

Comment:
This version check is just to confirm if everything is copacetic with ketch correct (set -e makes it so that if this command fails it exits)?

Reply:
Yeah - double checks on the off chance the install steps go fine but `ketch` is still missing for some reason.